auto_updater.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash set -eu
+
+function check_fork {
+  git ls-remote --exit-code fork > /dev/null 2>&1
+  if test $? = 1; then
+    git remote add fork git@github.com:adoptopenjdk-github-bot/homebrew-openjdk.git
+  fi
+}
+
+function check_hub {
+  command -v hub > /dev/null 2>&1
+  if test $? = 1; then
+    echo "Installing hub"
+    os=$(uname | awk '{print tolower($0)}')
+    wget --quiet "https://github.com/github/hub/releases/download/v2.12.3/hub-$os-amd64-2.12.3.tgz"
+    tar -xf hub-$os-amd64-2.12.3.tgz
+    export PATH="$PWD/hub-$os-amd64-2.12.3/bin:$PATH"
+    rm -rf hub-$os-amd64-2.12.3.tgz
+  fi
+  # Check that hub is in the PATH
+  which hub
+}
+
+check_fork
+check_hub
+
+cd Casks
+
+git fetch --all
+git reset --hard upstream/master
+
+for filename in *.rb; do
```

Comment:
Named function for this as well please :-)

Reply:
done